src/utils/getPreparedPeople.ts

```diff
@@ -0,0 +1,62 @@
+import { Person } from '../types';
+
+function filterByName(person: Person, query: string) {
+  const normalizedQuery = query.toLowerCase();
+
+  return person.name.toLowerCase().includes(normalizedQuery)
+    || person.fatherName?.toLowerCase().includes(normalizedQuery)
+    || person.motherName?.toLowerCase().includes(normalizedQuery);
+}
+
+function calculationCentry(year: number) {
+  return Math.ceil(year / 100);
+}
+
+export function getPreparedPeople(
+  people: Person[],
+  query: string,
+  sex: string | null,
+  centuries: string[],
+  sortField: string | null,
+  order: string | null,
+) {
+  let copyPeople = [...people];
```

Comment:
The `filter` method returns a copy of the array, so you don't need to do it manually.

Reply:
> The `filter` method returns a copy of the array, so you don't need to do it manually.

The same